TestingApp/.prettierrc.js

```diff
@@ -1,7 +0,0 @@
-module.exports = {
```

Comment:
you removed this, however there aren't any changes to the actual formatting of the code in the TestingApp. Are you sure this is the right thing to do? And run prettier?

Reply:
The test files in `TestingApp` have always been formatted and linted against the root ESLint and prettier, but these files had different rules than the root ones, which caused the editor to show a lot of formatting errors:

![image](https://github.com/supertokens/supertokens-react-native/assets/21277179/7a013ce5-219c-4091-97f4-72e18054dd9d)

I have also updated the prettier commands in `package.json` to format all js files in `TestingApp`